bazel/src/main/java/com/google/api/codegen/bazel/BazelBuildFileView.java

```diff
@@ -137,7 +137,9 @@ class BazelBuildFileView {
               // Default service name as it appears in the proto.
               : service;
       javaTests.add(javaPackage + "." + actualService + "ClientTest");
-      javaTests.add(javaPackage + "." + actualService + "ClientHttpJsonTest");
+      if ("grpc+rest".equals(transport)) {
```

Comment:
What if transport is only "rest"? Shouldn't we include the test then as well?

Reply:
If it is only rest, then the test is called just `ClientTest`. I.e. HttpJsonTests appear only if there is multitransport.